### PR TITLE
Only show current enrollments in Grades widget

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-13149
affects: student
release note: Fixed a bug that could cause assignments for concluded enrollments to show in the Grades widget

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Core/Core/API/APIRequestable.swift [hunk 1]
```diff
@@ -41,6 +42,8 @@ public enum APIQueryItem: Equatable {
             return APIQueryItem.array("include", includes).toURLQueryItems()
         case .perPage(let perPage):
             return APIQueryItem.value("per_page", String(perPage)).toURLQueryItems()
+        case .bool(let name, let value):
+            return APIQueryItem.value(name, value ? "1" : "0").toURLQueryItems()
```
Comment (bluwave) [inline Core/Core/API/APIRequestable.swift]: Do all canvas API's take Bool as a 1 or 0?  Or can it be "true" or "false" sometimes?

Comment (armstrongnate) [reply Core/Core/API/APIRequestable.swift]: It does support true/false as well.  Also yes/no, y/n, t/f, on/off. I figure we might as well be consistent.

Here's how the api parses boolean params:
```ruby
    def self.value_to_boolean(value, ignore_unrecognized: false)
      if value.is_a?(String) || value.is_a?(Symbol)
        return true if ["yes", "y", "true", "t", "on", "1"].include?(value.to_s.downcase)
        return false if ["no", "n", "false", "f", "off", "0"].include?(value.to_s.downcase)
      end
      return value if [true, false].include?(value)
      return nil if ignore_unrecognized
      return value.to_i != 0
    end
  end
```

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>



#### Release Note: 
Fixed a bug that could cause assignments for concluded enrollments to show in the Grades widget

#### Affected Apps: Student

[MBL-13149](https://instructure.atlassian.net/browse/MBL-13149)

<details><summary>:x: Swift lint</summary>


    ⚠️  /Users/vagrant/git/Core/CoreTests/API/APISubmissionRequestableTests.swift:106:52: Trailing Comma Violation: Multi-line collection literals should have trailing commas. (trailing_comma)
    Linting 'APIDashboardTests.swift' (470/667)
    Linting 'APIGroupRequestableTests.swift' (471/667)
</details>
<details><summary>:x: XCTest failed: CoreTests/APIRecentlyGradedRequestableTests/testQuery</summary>


    Assertion Failure: APIRecentlyGradedRequestableTests.swift:30: XCTAssertEqual failed: ("[Core.APIQueryItem.perPage(3), Core.APIQueryItem.include(["assignment"]), Core.APIQueryItem.bool("only_current_submissions", true)]") is not equal to ("[Core.APIQueryItem.value("per_page", "3"), Core.APIQueryItem.array("include", ["assignment"])]")
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/API/APIRequestable.swift (hunk 1 above)
Comment (bluwave) [reply Core/Core/API/APIRequestable.swift]: ah ok, yes this works well then.  good job.

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021443&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021445&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021441&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**